apps/backoffice-v2/src/pages/Entity/components/CaseCallToAction/CaseCallToAction.tsx

```diff
@@ -54,7 +54,7 @@ export const CaseCallToAction: FunctionComponent<ICaseCallToActionProps> = ({ va
         </DialogTrigger>
         <DialogContent className={`mb-96`}>
           <DialogHeader>
-            <DialogTitle className={`text-2xl`}>Ask for all re-uploads</DialogTitle>
+            <DialogTitle className={`text-2xl`}>Ask to re-upload</DialogTitle>
```

Comment:
It will change the title for all to re upload pop up window as well

Reply:
it wont brother, only this one